contracts/ANT-1/contract.ts

```diff
@@ -0,0 +1,46 @@
+// ~~ Put all the interactions from '../actions/` together to write the final handle function which will be exported
+// from the contract source. ~~
+import { balance } from './actions/read/balance';
+import { evolve } from './actions/write/evolve';
+import { removeController } from './actions/write/removeController';
+import { removeRecord } from './actions/write/removeRecord';
+import { setController } from './actions/write/setController';
+import { setName } from './actions/write/setName';
+import { setRecord } from './actions/write/setRecord';
+import { setTicker } from './actions/write/setTicker';
+import { transferTokens } from './actions/write/transferTokens';
+import { ANTState, ContractResult, PstAction } from './types';
+
+declare const ContractError;
```

Comment:
create `globals.d.ts` and add `ContractError` and `SmartWeave` similar to https://github.com/ar-io/testnet-contract/blob/develop/src/globals.d.ts and remove the declarations in each handler function